introduction/java_if_else/java_if_else.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+public class Solution {
+  
+  private static final String WEIRD = "Weird";
+  private static final String NOT_WEIRD = "Not Weird";
+  
+  private static String isWeird(int numberToCheck) {
+    if ((numberToCheck % 2 == 1) && (numberToCheck <= 100) && (numberToCheck >= 1)) {
+      return WEIRD; 
+    } else if((numberToCheck >= 2) && (numberToCheck <= 5)) { 
+        return NOT_WEIRD; 
+    } else if((numberToCheck >= 6) && (numberToCheck <= 20)) { 
+        return WEIRD; 
+    } else if((numberToCheck > 20) && (numberToCheck <= 100)) { 
+        return NOT_WEIRD; 
+    }
+    return NOT_WEIRD;
```

Comment:
I think that equal expression would be
`numberToCheck % 2 == 1 && numberToCheck <= 100 && numberToCheck >= 1 || numberToCheck >= 6 && numberToCheck <= 20`

So your function may look like this:

    bool isWeird(int value) {
      return value % 2 == 1 && value <= 100 && value >= 1 ||
          value >= 6 && value <= 20;
    }

It might be easier to read this way.